src/main/java/org/opensrp/repository/postgres/LocationRepositoryImpl.java

```diff
@@ -762,7 +762,18 @@ public int countSearchLocations(LocationSearchBean locationSearchBean) {
 	public List<com.ibm.fhir.model.resource.Location> findLocationByJurisdiction(String jurisdiction) {
 		return convertToFHIRLocation(findStructuresByProperties(false, jurisdiction, null));
 	}
-	
+
+
+	@Override
+	public List<com.ibm.fhir.model.resource.Location> findChildLocationByJurisdiction(boolean returnGeometry, String id) {
```

Comment:
should return list of String or Jurisdiction objects

Reply:
Returning a list o String